### PR TITLE
feat: update present example

### PR DESCRIPTION
**Pull request by lukasjhan**
After #138, I updated docs and examples for presentationFrame.

And fix some minor typos :)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -22,6 +22,28 @@ SD-JWT-JS is a [promise-based](https://javascript.info/promise-basics) SD JWT Cl
...
+```bash
+yarn add @sd-jwt/sd-jwt-vc
+```
+
+Using pnpm:
+
+```bash
+pnpm install @sd-jwt/sd-jwt-vc
+```
+
```
Comment (lukasjhan) [inline docs/README.md:44]: I added sd-jwt-vc install for credential use cases

File: docs/0.x/claims.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Claims
 
-You can get claims by using `getCliams` method
+You can get claims by using `getClaims` method
```
Comment (lukasjhan) [inline docs/0.x/claims.md:3]: fix typo

File: examples/sd-jwt-example/all.ts [hunk 3]
```diff
@@ -57,7 +57,7 @@ import { createSignerVerifier, digest, generateSalt } from './utils';
   console.log('encodedJwt:', credential);
 
   // Holder Receive the credential from the issuer and validate it
-  // Return a boolean result
+  // Return a result of header and payload
```
Comment (lukasjhan) [inline examples/sd-jwt-example/all.ts:60]: fix wrong comments

**Review by cre8 — APPROVED** [review]
LGTM

**Pull request merged**